PacketWrapper/src/main/java/com/comphenix/packetwrapper/AbstractPacket.java

```diff
@@ -2,24 +2,25 @@
  * PacketWrapper - ProtocolLib wrappers for Minecraft packets
  * Copyright (C) dmulloy2 <http://dmulloy2.net>
  * Copyright (C) Kristian S. Strangeland
- *
+ * <p>
```

Comment:
This comment is not treated as a Javadoc so instead of adding `<p>` tags I would suggest removing the second `*` from the comment start